lib/manager/gomod/readme.md

```diff
@@ -3,3 +3,6 @@ You might be interested in the following `postUpdateOptions`:
 1. `gomodTidy` - if you'd like Renovate to run `go mod tidy` after every update before raising the PR.
    1. This is implicitly enabled for major updates
 1. `gomodUpdateImportPaths` - if you'd like Renovate to update your source import paths on major updates before raising the PR.
+
+When Renovate is running using `binarySource=docker` (such as in the hosted WhiteSource Renovate app) then it will pick the latest compatible version of Go to run, i.e. the latest 1.x release.
```

Comment:
```suggestion
When Renovate is running using `binarySource=docker` (such as in the hosted WhiteSource Renovate app) then it will pick the latest compatible version of Go to run, i.e. the latest `1.x` release.
```

Maybe we want to use `monospaced` font for this version number as well?